infrastructure/pg/migrationhelper/filesystem/filesystem.go

```diff
@@ -0,0 +1,45 @@
+package filesystem
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/golang-migrate/migrate/v4"
+)
+
+type FilesystemMigrationHelper struct {
+	SourceURL   string
+	DatabaseURL string
+}
+
+func NewFilesystemMigrationHelper(sourceURL, databaeURL string) *FilesystemMigrationHelper {
+	return &FilesystemMigrationHelper{
+		SourceURL:   sourceURL,
+		DatabaseURL: databaeURL,
+	}
+}
+
+// Implement MigrationHelper interface
+func (fmh *FilesystemMigrationHelper) Migrate() {
+	if fmh.SourceURL == "" {
```

Comment:
Move validation to `NewFilesystemMigrationHelper`. And could we change `SourceURL` and `DatabaseURL ` to private?

Reply:
sure, will change it!